packages/cloud_defend/data_stream/alerts/manifest.yml

```
@@ -0,0 +1,13 @@
+title: "alerts"
+type: logs
+streams:
+  - input: logfile
+    title: Alerts
+    description: Alerted events
+    vars:
+      - name: paths
+        type: text
+        title: Paths
+        multi: true
+        default:
+          - /var/log/*.log
```

Should this block look like the below template?
```suggestion
  - input: cloudbeat/cis_k8s
    title: K8s CIS Compliance
    template_path: vanilla.yml.hbs
    description: |
      Check CIS Benchmark compliance
      ###### Only a single benchmark can be enabled. Default is K8s CIS.
```

---

This is a custom "integration input". Not entirely familiar with how this part all connects, but I imagine we'll need to create an input like "cloud_defend"  to relay the integration options to elastic-agent and then spin up the anduril service. https://github.com/elastic/anduril

I think we need to create a PR to elastic-agent repo and add a spec for anduril. e.g https://github.com/elastic/elastic-agent/blob/main/specs/cloudbeat.spec.yml. Not sure how the build artifact for anduril gets into elastic-agent though. (artifactory?)

https://github.com/elastic/integrations/blob/main/packages/cloud_security_posture/data_stream/findings/manifest.yml#L16
https://github.com/elastic/cloudbeat/blob/main/config/config.go#L41

Custom fleet integration UI can be done for a specific "input" too. e.g https://github.com/elastic/kibana/blob/4d99f0dd341d0c6bfc4387130b3872c302d5cd04/x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/eks_form.tsx#L23

@oren-zohar Are there any other pieces to the plumbing we should be aware of wrt to getting integration policy config into our new service?